pkg/util/utils.go

```diff
@@ -220,9 +223,43 @@ func UploadError(b []byte, url string) error {
 	return nil
 }
 
-func MaybeReportErrorAndExit(err error) {
+func MaybeReportErrorAndExit(errToReport error) {
+	var err error
 	if viper.GetBool(config.WantReportError) {
-		ReportError(err, reportingURL)
+		err = ReportError(errToReport, reportingURL)
+		if err != nil {
+			glog.Errorf(err.Error())
+		}
+	} else {
+		fmt.Println("An error has occurred.  Would you like to opt in to sending this and future crashes to minikube to help prevent future errors?")
+		if PromptUserForAccept() {
+			mConf, err := minikubeConfig.ReadConfig()
+			if err != nil {
+				glog.Errorf(err.Error())
+			}
+			minikubeConfig.SetBool(mConf, config.WantReportError, "true")
+			minikubeConfig.WriteConfig(mConf)
+			err = ReportError(errToReport, reportingURL)
+			if err != nil {
+				glog.Errorf(err.Error())
+			}
+		}
 	}
 	os.Exit(1)
 }
+
+func PromptUserForAccept() bool {
+	for {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print("Please enter your response [Y/N]: ")
```

Comment:
What is meant by default?  I enforced that the user responds.  Should I instead make it so that no response (return) is an acceptance ('Y')?